### DISCUSSION TITLE
Multi-colored text with Qterminal

### DISCUSSION THREAD

**Discussion by edwardp3** [lxqt/lxqt#2378] [Q&A] [answered]
https://9to5linux.com/sparkylinux-6-6-brings-latest-debian-bullseye-updates-linux-6-1-kernel-support has a graphic of Qterminal, displaying multi-colored text going across its window.

Is there a setting in Qterminal for this? 

Thanks in advance.

**Comment by eriefisher** [comment, marked as answer]
Look at his input. He piped neofetch to lolcat. That's what creates the colours. Install lolcat if you don't already have it installed.

![screen](https://user-images.githubusercontent.com/567829/217352857-77f54469-02c2-45b6-8977-de16eec45f61.jpg)

**Reply by edwardp3** [reply]
I didn't notice lolcat at first. That's cool!
